WordPress/Classes/ViewRelated/Media/Tenor/TenorResponse.swift

```diff
@@ -0,0 +1,166 @@
+import Foundation
+import MobileCoreServices
+
+typealias TenorGifFormat = TenorMedia.MediaCodingKeys
+
+class TenorResponse: NSObject, Decodable {
+    let next: String
+    let results: [TenorMedia]
+}
+
+class TenorMedia: NSObject, Decodable {
+    enum CodingKeys: String, CodingKey {
+        case id
+        case created
+        case itemurl
+        case title
+        case media
+    }
+
+    enum MediaCodingKeys: String, CodingKey, CaseIterable {
+        case nanogif
+        case tinygif
+        case mediumgif
+        case gif
+    }
+
+    let id: String
+    let created: Date
+    let itemurl: String
```

Comment:
Do you think we can get rid of `created` and `itemurl`?

Reply:
`created` is used for `WPMediaAsset` conformance, but `itemurl` seems to be a leftover. I'll remove it.